test/UnitTest/v1/test_casesNGSIv1.py

```diff
@@ -16,704 +16,974 @@
         url=brokerIp+"/ngsi10/queryContext"
         headers={'Content-Type':'appliction/json'}
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata42),headers=headers)
-        #print(r.status_code)
-        #print(r.content)
+        print(r.content)
         assert r.status_code == 200
 
 #testCase 22
 '''
-  Testing  for query request using type
+  To test  for query request using type
 '''
 def test_queryrequest2():
         url=brokerIp+"/ngsi10/queryContext"
         headers={'Content-Type':'appliction/json'}
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata43),headers=headers)
-        #print(r.status_code)
-        #print(r.content)
+        print(r.content)
         assert r.status_code == 200
 
+
 #testCase 23
 '''
-  Testing  for query request using geo-scope(polygon)
+  To test  for query request using geo-scope(polygon)
 '''
 def test_queryrequest3():
         url=brokerIp+"/ngsi10/queryContext"
         headers={'Content-Type':'appliction/json'}
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata44),headers=headers)
-        #print(r.status_code)
-        #print(r.content)
+        print(r.content)
         assert r.status_code == 200
-	
+
 #testCase 24
 '''
-  Testing  for query request multiple filter
+  To test  for query request multiple filter
 '''
 def test_queryrequest4():
         url=brokerIp+"/ngsi10/queryContext"
         headers={'Content-Type':'appliction/json'}
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata45),headers=headers)
-        #print(r.status_code)
-        #print(r.content)
+        print(r.content)
         assert r.status_code == 200
+
+#testCase 25
+'''
+  To test if wrong payload is decoded or not
+'''
+def test_case25():
+        url=brokerIp+"/ngsi10/updateContext"
+        headers={'Content-Type':'appliction/json'}
+        r=requests.post(url,data=json.dumps(data_ngsi10.subdata46),headers=headers)
+        print(r.status_code)
+        assert r.status_code == 500
+
+
+#testCase26
+'''
+  To test the response on passing DELETE in updateAction in payload
+'''
+def test_case26():
+        #create v1 entity
+        url=brokerIp+"/ngsi10/updateContext"
+        headers={'Content-Type':'appliction/json','User-Agent':'lightweight-iot-broker'}
+        r=requests.post(url,data=json.dumps(data_ngsi10.subdata47),headers=headers)
+        print(r.content)
+
+        #get the created entity
+        url=brokerIp+"/ngsi10/entity/Result047"
+        r=requests.get(url)
+	print(r.content)	
+
+        #passing DELETE in update payload
+        url=brokerIp+"/ngsi10/updateContext"
+        headers={'Content-Type':'appliction/json','User-Agent':'lightweight-iot-broker'}
+        r=requests.post(url,data=json.dumps(data_ngsi10.subdata48),headers=headers)
+        print(r.content)
+
+        #get the created entity
+        url=brokerIp+"/ngsi10/entity/Result047"
+        r=requests.get(url)
+        print(r.content)
+        assert r.status_code == 404
+
```

Comment:
For test case #27, #28, #29: correction is required, empty payload corresponds to subdata49, not subdata48.